src/components/ContextMenuItem.tsx

```diff
@@ -101,23 +92,12 @@ function ContextMenuItem({onPress, successIcon, successText, icon, text, isMini,
             style={StyleUtils.getContextMenuItemStyles(windowWidth)}
             isAnonymousAction={isAnonymousAction}
             focused={isFocused}
-            interactive={isThrottledButtonActive}
+            interactive
+            iconType={CONST.ICON_TYPE_ICON}
```

Comment:
In one of my [PRs](https://github.com/Expensify/App/pull/33242/files#diff-9db0c2baf85f38b9a238d03f1089a686039ab6c6b94ef412bda920293909b7a4) I made iconType to be optional, what do you think?

Reply:
Great idea! I will do the same 😄